tests/console/check_system_info.pm

```diff
@@ -81,6 +81,16 @@ sub check_buildid {
     }
 }
 
+# SLE-21916: change bzr to breezy
+# check in the upgraded sysetem that bzr was repalced by breezy
```

Comment:
I think you'd better add test steps in function description so that to make reviewers to understand your test better.

Reply:
added